app/CommandHandling/CommandBusServiceProvider.php

```diff
@@ -216,31 +219,44 @@ function (): CompositeCommandValidator {
             }
         );
 
-        $container->add(
-            'resque_command_bus_factory',
-            new ObjectArgument(
-                function ($queueName) use ($container) {
-                    $container->addShared(
-                        $queueName . '_command_bus',
-                        function () use ($queueName, $container) {
-                            return new ValidatingCommandBusDecorator(
-                                new ContextDecoratedCommandBus(
-                                    self::createResqueCommandBus($queueName, $container),
-                                    $container
-                                ),
-                                new CompositeCommandValidator()
-                            );
-                        }
-                    );
+        $container->addShared(
```

Comment:
Like this approach a lot more than the looping mechanism. It's a lot more explicit and thus readable imo.